src/platform/xcode_templates.nit

```diff
@@ -528,7 +496,7 @@ class PlistTemplate
 	<key>CFBundleShortVersionString</key>
 	<string>{{{short_version}}}</string>
 	<key>CFBundleSignature</key>
-	<string>\\?\\?\\?\\?</string>
+	<string>?{{{"??"}}}?</string>
```

Comment:
It is the only way I found the avoid the C compiler warning on `????`. The previous way didn't work, it resulted in `\?\?\?\?` and Xcode warned that it was an invalid value.